### PR TITLE
Battery : Redesign BatteryMeterView

### PR DESCRIPTION
**Pull request by joepacino**
Change-Id: Iaa5e0be90f03034401fb341519df7cbab1edbb08
Signed-off-by: IacobIonut01 <paulionut2003@gmail.com>

### CODE REVIEW

File: res/layout/battery_header.xml [hunk 1]
```diff
@@ -17,6 +17,7 @@
 
 <LinearLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
+	xmlns:app="http://schemas.android.com/apk/res-auto"
```
Comment (gwolf2u) [inline res/layout/battery_header.xml]: try to correct spacing here

File: res/layout/battery_header.xml [hunk 2]
```diff
@@ -31,7 +32,8 @@
         android:layout_height="wrap_content"
         android:layout_weight="1"
         android:layout_marginStart="56dp"
-        android:orientation="vertical">
+        android:orientation="vertical"
+		android:visibility="gone">
```
Comment (gwolf2u) [inline res/layout/battery_header.xml]: same here

File: res/layout/battery_header.xml [hunk 3]
```diff
@@ -56,11 +58,21 @@
...
+        android:background="@drawable/battery_metter_gauage"
+        android:padding="4dp">
 
-    <com.android.settings.fuelgauge.BatteryMeterView
-        android:id="@+id/battery_header_icon"
-        android:layout_width="@dimen/battery_meter_width"
-        android:layout_height="@dimen/battery_meter_height"
-        android:layout_marginEnd="16dp"/>
+        <com.android.settings.fuelgauge.BatteryMeterView
+			android:id="@+id/battery_header_icon"
```
Comment (gwolf2u) [inline res/layout/battery_header.xml]: and here

**Review by gwolf2u — CHANGES_REQUESTED** [review]
just correct some spacing issues

**Comment by joepacino** [comment]
fix spaces

**Comment by gwolf2u** [comment]
fixed some leftover spacing myself 
also merged in https://github.com/crdroidandroid/android_packages_apps_Settings/commit/4df5a24a30329418ce6507e804d423ac21738f6e

**Pull request closed without merging**